### DISCUSSION TITLE
Highlight current track in list

### DISCUSSION THREAD

**Discussion by neoncobra** [losnoco/Cog#354] [Q&A]
Hi! Please tell me how can I highlight currently playing track in list. Now are all the same color, it hard to find which one is now playing

**Comment by kode54** [comment]
Command+L, Select Currently Playing, in the Edit menu. I can see why this may be confusing, since I had to spend a full minute to find it. There's also a column enabled by default which should be showing a status icon in the playlist, next to playing and queued tracks.
